README.md

```diff
@@ -27,16 +27,31 @@ Or install it yourself as:
       environment: "production",
       application: "ror"
     }
-    logger = Hivelog::Logger.new(:stdout, @labels)
+
     options = {
       event: event,
       user: user,
       group: group,
       tags: tags
     }
-    logger.debug("deb ya", options)
+
+    Hivelogger = Hivelog::Logger.new(:stdout, @labels)
+
+    Hivelogger.debug("deb ya", options)
+    Hivelogger.warn("warning message", options)
+    Hivelogger.info("information message", options)
+    Hivelogger.error("error message", options)
 ```
 
+Each log is a sentence. A sentence always starts with capitalization and ends with a period.
+
+Please scope the service with `tags` (e.g.: `Donations`, `NetworkEvents`, `Recurly`, ...). It will help for filtering.
```

Comment:
Maybe we can say to add a `service` key to the options hash with the service name (same as the one used for Gettext for exemple, eg: `Hivelogger.info("information message", service: 'Recurly')`)

Reply:
Something like that ? 
```
    options = {
      service: 'webapp',
      module: 'topics',
      tags: ['memberships', 'payment'],
    }
    Hivelogger.info("information message", options)
```